app/admin/user.rb

```diff
@@ -196,7 +196,7 @@
 
     f.inputs I18n.t('active_admin.user.roles') do
       f.has_many :user_rights, allow_destroy: true, new_record: true do |ur|
-        ur.input :right, as: :select, collection: UserRight.human_attribute_values(:rights, raw_values: true).invert.to_a, include_blank: false
+        ur.input :category, as: :select, collection: UserRight.categories.keys.map{ |cat| [I18n.t("activerecord.attributes.user_right/categories.#{cat}"), cat] }, include_blank: false
```

Comment:
```suggestion
        ur.input :category, as: :select, collection: UserRight.categories.keys.map{ |cat| [I18n.t(cat, scope: "activerecord.attributes.user_right/categories"), cat] }, include_blank: false
```
J'ai souvenir d'avoir lu une "good practice" qu'il fallait éviter l'interpolation dans les clés i18n mais impossible de retrouver où.

Reply:
yep, t'as raison